src/Nethermind/Nethermind.Blockchain/Receipts/IReceiptConfig.cs

```diff
@@ -21,4 +21,7 @@ public interface IReceiptConfig : IConfig
 
     [ConfigItem(Description = "Number of recent blocks to maintain transaction index. 0 to never remove tx index. -1 to never index.", DefaultValue = "2350000")]
     long? TxLookupLimit { get; set; }
+
+    [ConfigItem(Description = "Max num of block per eth_getLogs request.", DefaultValue = "10000")]
```

Comment:
As it doesn't matter with Bloom Index (which is default), maybe we should hide it from docs?